diambra/arena/stable_baselines3/make_sb3_env.py

```diff
@@ -1,53 +1,54 @@
 import os
-import sys
+import time
 import diambra.arena
 
+from stable_baselines3.common.monitor import Monitor
 from stable_baselines3.common.vec_env import DummyVecEnv, SubprocVecEnv
 from stable_baselines3.common.utils import set_random_seed
-from stable_baselines3.common.monitor import Monitor
 
-# Make Stable Baselines Env function
+# Make Stable Baselines3 Env function
 def make_sb3_env(game_id: str, env_settings: dict={}, wrappers_settings: dict={},
-                 use_subprocess: bool=True, seed: int=0, log_dir_base: str="/tmp/DIAMBRALog/",
-                 start_index: int=0, allow_early_resets: bool=True,
-                 start_method: str=None, no_vec: bool=False):
+                 episode_recording_settings: dict={}, render_mode: str="rgb_array", seed: int=None,
+                 start_index: int=0, allow_early_resets: bool=True, start_method: str=None,
+                 no_vec: bool=False, use_subprocess: bool=True, log_dir_base: str="/tmp/DIAMBRALog/"):
     """
     Create a wrapped, monitored VecEnv.
     :param game_id: (str) the game environment ID
     :param env_settings: (dict) parameters for DIAMBRA Arena environment
-    :param wrappers_settings: (dict) parameters for environment
-                              wraping function
+    :param wrappers_settings: (dict) parameters for environment wrapping function
+    :param episode_recording_settings: (dict) parameters for environment recording wrapping function
     :param start_index: (int) start rank index
     :param allow_early_resets: (bool) allows early reset of the environment
-    :param start_method: (str) method used to start the subprocesses.
-                        See SubprocVecEnv doc for more information
-    :param use_subprocess: (bool) Whether to use `SubprocVecEnv` or
-                          `DummyVecEnv` when
-    :param no_vec: (bool) Whether to avoid usage of Vectorized Env or not.
-                   Default: False
-    :param seed: (int) initial seed for RNG
+    :param start_method: (str) method used to start the subprocesses. See SubprocVecEnv doc for more information
+    :param use_subprocess: (bool) Whether to use `SubprocVecEnv` or `DummyVecEnv`
+    :param no_vec: (bool) Whether to avoid usage of Vectorized Env or not. Default: False
     :return: (VecEnv) The diambra environment
     """
 
     env_addresses = os.getenv("DIAMBRA_ENVS", "").split()
     if len(env_addresses) == 0:
         raise Exception("ERROR: Running script without DIAMBRA CLI.")
-        sys.exit(1)
 
     num_envs = len(env_addresses)
 
+    # Seed management
```

Comment:
I definitely see your point, the reason why I added this is because I want to make sure the seed is aligned in all the components/modules we use, so that replicability can be achieved (to the maximum extent possible, the emulator has its own RNG that is out of our control.....). So I added this check here to make sure the seed passed to stable baselines 3 (see later line 50) is the very same we use in the environment settings, you see what I mean?

Reply:
Ah got it, makes sense.